src/command/format.rs

```diff
@@ -49,6 +52,25 @@ pub fn format_cmd(
         paths
     );
 
+    let client = match Connector::new().connect().await {
+        Err(e) => {
+            warn!(
+                "watchman is not available (err = {:?}), falling back on stat(2)",
+                e
+            );
```

Comment:
Most users should be able to use treefmt without `watchman`, so having this as a warning isn't the best user experience for them.

I propose three modes: "auto" (default) tries watchman and silently falls back on "stat," "stat" is stat, and "watchman" errors if watchman is not available.

Then, introduce a CLI flag and/or project-level global config to allow switching between modes.